### DISCUSSION TITLE
Blank page, can't install

### DISCUSSION THREAD

**Discussion by pihokkerr** [YOURLS/YOURLS#2873] [Q&A] [answered]
Hi! 

System:
Synology DS218 NAS
MySQL-/MariaDB and phpMyAdmin
Virtual Host Web Server with my own domain on NAS (Nginx and PHP 7.2)
At the moment I have a functioning Wordpress website I have no problem with it at all. 

Today I tried to install YOURLS (I followed this video tutorial: https://youtu.be/BYCrtyH6xNk) but I can't access the page where I could install Yourls. I copied the contents of the YOURLS folder to web/s/ folder. 
https://mypage.com - I can see my Wordpress page.
https://mypage.com/s/ - 403 Error
https://mypage.com/s/admin/ - Blank page, no error, nothing

I tried YOURLS-1.8.1 and YOURLS-1.7.9 also.

What can be the problem?

Thanks for your help!

**Comment by dgw** [comment, marked as answer]
Please see https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps, especially the part about checking server logs.
